compiler/control/OMROptions.cpp

```diff
@@ -1059,9 +1059,9 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
    {"scount=",            "O<nnn>\tnumber of invocations before loading relocatable method in shared cache",
         TR::Options::setCount, offsetof(OMR::Options,_initialSCount), 1, "F%d"},
    {"scratchSpaceLimit=",    "C<nnn>\ttotal heap and stack memory limit, in KB",
-                                         TR::Options::setStaticNumericKBAdjusted, (intptr_t)&OMR::Options::_scratchSpaceLimit, 0, "F%d (KB)"},
+                                         TR::Options::setStaticNumericKBAdjusted, (intptr_t)&OMR::Options::_scratchSpaceLimit, 0, "F%d"},
```

Comment:
Would it be better to say "bytes" here for clarity then since it is different from the units they would have set the value in the first place?

Reply:
I updated as suggested.